config/initializers/sentry.rb

```diff
@@ -0,0 +1,13 @@
+##
+# * https://sentry.io/for/rails/
+# * https://github.com/getsentry/raven-ruby
+#
+Raven.configure do |config|
+  # Sentry will be enabled if SENTRY_DSN exists. Sentry reporting will work if
+  # SENTRY_DSN has a meaningful value.
+  config.dsn = ENV["SENTRY_DSN"]
+
+  # We expect SENTRY_ENV to always be set so fail with an error if it is
+  # missing
+  config.current_environment = ENV.fetch("SENTRY_ENV")
```

Comment:
For Harbours, I had this default to Rails.env, so if you already had your rails env configured matching up to what you wanted, then this didn't need to be defined as well. I introduced this override just for applications that needed to report errors in a different env to what they were running in. 

The additional benefit to doing `ENV.fetch("SENTRY_ENV", Rails.env)` here is that you wouldn't have that issue with .env needing to be created before webpacker could run.

Reply:
```suggestion
  config.current_environment = ENV.fetch("SENTRY_ENV", Rails.env)
```

Reply:
The JS side isn't using webpack ERB loader so I'll have to add a fallback to that too. Will do now.